Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
Here is the thing: this bundle says it is compatible with both versions 2 AND 3.

Your final code should look like this : 

```php
if (Kernel::VERSION_ID <= 20800) {
    // NEXT_MAJOR remove this block
    $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
} else {
    $media->setProviderName($this->getRequest()->get($this->getUniqid(), null, true)['providerName'])
}
```